app/src/main/AndroidManifest.xml

```diff
@@ -34,7 +34,6 @@
             android:theme="@android:style/Theme.Translucent.NoTitleBar" />
         <activity
             android:name=".LoginActivity"
-            android:noHistory="true"
```

Comment:
This causes issues with fb login in some devices, hence removed.
Refer: http://stackoverflow.com/questions/13850419/android-sso-login-dialog-disappears-immediately-with-no-errors